src/Audit.php

```diff
@@ -179,4 +179,80 @@ public function getModified($json = false, $options = 0, $depth = 512)
 
         return $json ? json_encode($modified, $options, $depth) : $modified;
     }
+
+    /**
+     * @return return all audits that were related to $this
+     */
+    public function getRelatedAudits()
```

Comment:
Christ, you managed to use `camelCase`, `snake_case` and `PascalCase` in the same file!

Reply:
That was unnecessary. If there is a convention you want me to follow, please let me know and I will happily conform..

Reply:
Like pretty much everybody else, we conform to PSR-2. And while not explicit, if you conform to the style of the existing code, you'll be able to figure that out.

Reply:
BTW, our in house convention is non-objects (ints, strings, bools) are snake case, methods in camelcase and objects in Pascal case.

Reply:
This is a different house, innit.